File: SEP3_T2/grpc-springboot/src/main/java/sep3/dto/CreatePostDTO.java
package sep3.dto;

public class CreatePostDTO {
    private String title;
    private String content;
    private int accountId;
    private int categoryId;

    public CreatePostDTO(String title, String content, int accountId) {
        this.title = title;
        this.content = content;
        this.accountId = accountId;
    }

    public String getTitle() {
        return title;
    }

    public String getContent() {
        return content;
    }

    public int getAccountId() {
        return accountId;
    }

    public int getCategoryId() {
        return categoryId;
    }
}

File: SEP3_T1/Shared/Domain/DTOs/CreatePostDTO.cs
namespace DTOs.User.PostDTOs;

public class CreatePostDTO(string title, string content, int categoryId, int accountId)
{
    public string Title { get; set; } = title;
    public string Content { get; set; } = content;
    public int CategoryId { get; set; } = categoryId;
    public int AccountId { get; set; } = accountId;
}